addons/mail/static/src/core/common/chat_window.js

```diff
@@ -112,12 +128,17 @@ export class ChatWindow extends Component {
         this.chatWindowService.close(this.props.chatWindow, options);
     }
 
-    get moreMenuText() {
-        return _t("More actions");
+    get OPEN_ACTIONS_MENU() {
+        return _t("Open Actions Menu");
+    }
```

Comment:
I believe that in recent code we no longer name getters for translated content like this. Instead we'd use “actionsMenuTitleText” or something like that

Reply:
I much prefer `OPEN_ACTIONS_MENU` over `actionsMenuTitleText`.
It's kinda wysiwyg, which drastically improves template readability, And the more you don't have to find a new name for something, the better.

Reply:
Do we have a fixed rule for naming this?

Reply:
There's no rule, so it's more about making the code as readable and maintainable as possible. Sometimes when you can afford to not make a new name / concept, then you help readability / maintainability of the code, because it's easier to understand a codebase with 10 concepts than a equivalent codebase with 100 concepts.